src/battle_ai_switch_items.c

```diff
@@ -65,30 +65,16 @@ void GetAIPartyIndexes(u32 battler, s32 *firstId, s32 *lastId)
     }
 }
 
-bool32 IsSwitchinIdValid(u32 battler, u32 switchinId)
+bool32 SetSwitchinAndSwitch(u32 battler, u32 switchinId)
```

Comment:
does this need to be a function? If you want it to keep it a function it should probably be a static inline

Reply:
It doesn't need to be after moving the bugfix, but I'd prefer we keep it one, cuz I like
```
return SetSwitchinAndSwitch(battler, PARTY_SIZE);
```
a lot better than
```
  {
      gBattleStruct->AI_monToSwitchIntoId[battler] = switchinId;
      return TRUE;
  }
```
cuz it's shorter and faster to write. If I get around to refactoring this into one big function instead of several little ones I'd remove it then but I'd prefer to keep it for now if that's cool :)

Reply:
That's fair but I think it could be a simple static inline then, right? Also capitalize `i`

Reply:
Yeah totally am in the middle of pushing that! :D 

You mean `SetSwitchInAndSwitch`? Most of the code I've written refers to the incoming mon as a "Switchin" so I'd prefer to keep it decapped if that's the `i` you're talking about, ie `switchinCandidate`

Reply:
ah, okay